third-party/dnd-wiki/classes/class-weapon-master.xml

```diff
@@ -0,0 +1,454 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<elements>
+	<info>
+		<name>Weapon Master</name>
+		<description>The Weapon Master class.
+</description>
+		<author
+url="https://www.dandwiki.com/wiki/Weapon_Master_(5e_Class)">Blobby383b and Raiden</author>
+		<update version="1.0">
+			<file name="class-weapon-master.xml" url="https://raw.githubusercontent.com/aurorabuilder/elements/master/third-party/dnd-wiki/classes/class-weapon-master.xml" />
+		</update>
+	</info>
+	<element name="Weapon Master" type="Class" source="D&amp;D Wiki" id="ID_WIKI_CLASS_WEAPON_MASTER">
+		<description>
+			<p>A battle rages on the plains, the orcs slowly pushing the empire's soldiers back foot by foot. Suddenly a lone warrior rushes into the fray from the side, cutting a swath of death and gore into the orcish lines. The warrior, rallying the troops, leads them to victory.</p>
+			<h4>THE LEGEND</h4>
+			<p>The Weapon masters are warriors whose legendary status has been maintained throughout the years, and whose legendary exploits could fill enough books to satisfy even the greatest of libraries. The Weapon master is a skillful fighter who can use any weapon to its utmost level of perfection and treat it as if it were an extension of their body, enhancing their natural combat prowess beyond the limits of a normal soldier.</p>
+			<h4>UNSEEN VICTORY</h4>
+			<p>The Goliath and the Gnome are alone in the arena, ready for the final fight of the tournament. Everyone knows who the winner will be, but the fight starts anyway. The Goliath swings its club at the poor creature, only to find that he isn't there. The gnome rushes the Goliath's legs, and starts cutting with unimaginable speed, until the giant can no longer stand. Pressing his glaive to his opponent's neck, the judge calls the fight.</p>
+			<h4>CREATING A WEAPON MASTER</h4>
+			<p></p>
+			<h5>QUICK BUILD</h5>
+			<p>You can make a Weapon master quickly by following these suggestions. First, Strength or Dexterity should be your highest ability score, followed by Constitution. Second, choose the Soldier background.</p>
+			<h3>CLASS FEATURES</h3>
+			<p>As a Weapon Master you gain the following class features.</p>
+			<h5>HIT POINTS</h5>
+			<p>
+				<span class="emphasis">Hit Dice:</span>1d10 per Weapon Master level
+				<br />
+				<span class="emphasis">Hit Points at 1st Level:</span>10 + your Constitution modefier
+				<br />
+				<span class="emphasis">Hit Points at Higher Levels:</span>1d10 (or 6) + your Constitution modifier per Weapon Master level after 1st
+			</p>
+			<h5>PROFICIENCIES</h5>
+			<p>
+				<span class="emphasis">Armor:</span>All armor,  shields 
+				<br />
+				<span class="emphasis">Weapons:</span>Simple weapons, martial weapons 
+				<br />
+				<span class="emphasis">Tools:</span>Smith's tools
```

Comment:
The proficiency for smith's tools isn't granted in the rules.

Reply:
Added.